### PR TITLE
[7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by opeyemipeter1759**
## Task: `Name of given task`
- `Changes you made to given task`
image

### CODE REVIEW

File: submissions/opeyemiPeter/htmlForms/signupPage.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+             
+              <select name="Select a country" id="country" >
+                <option value="Nigeria">Nigeria</option>
+                <option value="Nigeria">Ghana</option>
+                <option value="Nigeria">South Afica</option>
+                <option value="Nigeria">Kenya</option>
+                <option value="Nigeria">Others</option>
+                 
+              </select>
+
```
Comment (adaugochi) [inline submissions/opeyemiPeter/htmlForms/signupPage.html:49]: Remove spaces from line 49-51

File: submissions/opeyemiPeter/htmlForms/signupPage.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                <option value="Nigeria">Others</option>
+                 
+              </select>
+              
+             
+            
+            
+              <button type="submit">Sign up for free</button><br> <br>
+            </div> <br>
+
```
Comment (adaugochi) [inline submissions/opeyemiPeter/htmlForms/signupPage.html:55]: remove spaces

File: submissions/opeyemiPeter/htmlForms/signupPage.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+             
+              <input type="checkbox" checked="checked" name="remember" id="checkbox-signup"><a href="#"> I agree to the Terms and Conditions</a>
+              </label><br>
```
Comment (adaugochi) [inline submissions/opeyemiPeter/htmlForms/signupPage.html:58]: remove space from line 59 - 62

File: submissions/opeyemiPeter/htmlForms/signupPage.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (adaugochi) [inline submissions/opeyemiPeter/htmlForms/signupPage.html:2]: Task 7 doesn't require you to create `signUp` page. What you are asked to do is to create a contact form page. Follow the instruction given. Here is a link to the task https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/24

**Comment by opeyemipeter1759** [comment]
@adaugochi kindly look into my PR

File: submissions/opeyemiPeter/htmlForms/signupPage.html (hunk 4 above)
Comment (opeyemipeter1759) [reply submissions/opeyemiPeter/htmlForms/signupPage.html:2]: how do i remove the sign up page. thanks

Comment (opeyemipeter1759) [reply submissions/opeyemiPeter/htmlForms/signupPage.html]: @adaugochi i have tried removing the signup page.
but  do i have to wait till this is merged before pushing the signup page?

**Pull request merged**